tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -8,7 +8,17 @@ ifneq ($(filter cmsis-nn,$(ALL_TAGS)),)
     THIRD_PARTY_DOWNLOADS += \
       $(eval $(call add_third_party_download,$(CMSIS_URL),$(CMSIS_MD5),cmsis,patch_cmsis))
 
-    CMSIS_PATH = $(MAKEFILE_DIR)/downloads/cmsis/
+    ifeq ($(CMSIS_PATH),)
+      CMSIS_PATH = $(MAKEFILE_DIR)/downloads/cmsis/
```

Comment:
let's also add to the readme some of the various ways in which this particular target can be invoked, e.g. with an unpatched CMSIS outside of the TF Micro tree ...

Reply:
Done